packages/contract-artifacts/package.json

```diff
@@ -10,13 +10,19 @@
         "test": "test"
     },
     "scripts": {
+        "artifacts_copy": "node ../monorepo-scripts/lib/contract_artifacts_copy.js ${npm_package_config_contractsPackages}",
+        "artifacts_transform": "node ../monorepo-scripts/lib/contract_artifacts_transform.js ./artifacts && prettier --write ./artifacts/*.json &&  cp -r ./artifacts/ ../../python-packages/contract_artifacts/src/zero_ex/contract_artifacts/artifacts/",
+        "artifacts_update": "yarn artifacts_copy && yarn artifacts_transform",
         "build": "yarn tsc -b",
         "build:ci": "yarn build",
         "test": "yarn run_mocha",
         "test:circleci": "yarn test",
         "run_mocha": "mocha --require source-map-support/register --require make-promises-safe lib/test/**/*_test.js --bail --exit",
         "clean": "shx rm -rf lib"
     },
+    "config": {
+        "contractsPackages": "@0x/contracts-asset-proxy @0x/contracts-erc20 @0x/contracts-erc721 @0x/contracts-erc1155 @0x/contracts-exchange @0x/contracts-exchange-forwarder @0x/contracts-exchange-libs @0x/contracts-extensions @0x/contracts-multisig @0x/contracts-test-utils @0x/contracts-utils @0x/contracts-coordinator"
```

Comment:
In the script, let's actually read it from the root `package.json` instead of duplicating this here.

Reply:
To get this to build, I had to change `rootDir: ./` to `rootDir: ../..` in the `tsconfig.json`. Waiting for CI to see if it causes any issues.

Reply:
This wound up breaking relative links by introducing nesting to the build output. I'm not sure it's worth it to introduce this weirdness here. 

Build output:
```
lib
├── package.json
└── packages
    └── contract-artifacts
        ├── artifacts
        │   ├── AssetProxyOwner.json
        │   ├── Coordinator.json
        │   ├── CoordinatorRegistry.json
        │   ├── DummyERC20Token.json
        │   ├── DummyERC721Token.json
        │   ├── DutchAuction.json
        │   ├── ERC20Proxy.json
        │   ├── ERC20Token.json
        │   ├── ERC721Proxy.json
        │   ├── ERC721Token.json
        │   ├── Exchange.json
        │   ├── Forwarder.json
        │   ├── IAssetProxy.json
        │   ├── IValidator.json
        │   ├── IWallet.json
        │   ├── MultiAssetProxy.json
        │   ├── OrderValidator.json
        │   ├── WETH9.json
        │   └── ZRXToken.json
        ├── src
        │   ├── copy.d.ts
        │   ├── copy.d.ts.map
        │   ├── copy.js
        │   ├── copy.js.map
        │   ├── index.d.ts
        │   ├── index.d.ts.map
        │   ├── index.js
        │   ├── index.js.map
        │   ├── transform.d.ts
        │   ├── transform.d.ts.map
        │   ├── transform.js
        │   └── transform.js.map
        └── test
            ├── contract_artifacts_test.d.ts
            ├── contract_artifacts_test.d.ts.map
            ├── contract_artifacts_test.js
            └── contract_artifacts_test.js.map
```

Reply:
Let's use `fs.readFile`